packages/eas-build-job/src/android.ts

```diff
@@ -175,4 +178,6 @@ export const JobSchema = Joi.object({
     submitProfile: Joi.string(),
   }),
   loggerLevel: Joi.string().valid(...Object.values(LoggerLevel)),
+
+  workflowInterpolationContext: Joi.any(),
```

Comment:
shouldn't it be never for standard builds?

Reply:
Hmm, yea, I guess, for clarity we can make it so. 👍